.travis.yml

```diff
@@ -1,7 +1,8 @@
-
 jobs:
   include:
-    - name: Unit Tests
+    - stage: "Tests"
+      name: "Unit Tests"
```

Comment:
They're not but I like to use them to highlight labels (only description text or name)

Reply:
nitpicking, but I agree with @ThomasF34 : strings should be enclosed the same everywhere to respect the principle of less surprise and the obvious best option for this to me is to enclose them only when needed to avoid having to escape them.